src/ar/arabshentai/src/eu/kanade/tachiyomi/extension/ar/arabshentai/ArabsHentai.kt

```diff
@@ -54,7 +54,7 @@ class ArabsHentai : ParsedHttpSource() {
             thumbnail_url = element.selectFirst("a .poster img")?.imgAttr()
         }
 
-    override fun popularMangaNextPageSelector() = ".pagination a.arrow_pag i#nextpagination"
+    override fun popularMangaNextPageSelector() = ".pagination a#nextpagination"
```

Comment:
This selector doesn't work. Have you tried with:
```suggestion
    override fun popularMangaNextPageSelector() = ".pagination > .current + .inactive"
```

---

These is also a 301 redirect on browse. Can you skip this redirect page in the extension?
<img width="470" height="94" alt="image" src="https://github.com/user-attachments/assets/6a7011fe-f58c-46d4-b110-4eb4126537a4" />

Reply:
Yes, it is possible to direct some chapters. It is not possible to direct. The chapter format has changed. It appears when you open the chapter. Numbers appear next to the chapter number.